docker/Dockerfile

```diff
@@ -149,6 +149,23 @@ RUN --mount=type=ssh \
 # Setup container for runtime environment
 FROM conda_env as runtime
 
+ARG MORPHEUS_USER="root"
+
+# Conditionally create user and install sudo
+RUN if [ ! -z "${MORPHEUS_USER}" ] && [ "${MORPHEUS_USER}" != "root" ] ; then \
+    useradd -ms /workspace ${MORPHEUS_USER} \
+      && usermod -aG sudo ${MORPHEUS_USER} \
+      && apt-get update \
+      && apt-get install -y sudo \
+      && apt-get clean \
+      && echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers ; \
+    else \
+      echo "Skipping user creation..." ;\
+    fi
+
+# Allow default user to execute sudo commands in container
+RUN echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
```

Comment:
Shouldnt this go in the `else` branch of the above if statement? Instead of the line `echo "Skipping user creation..." ;\`. This would save a layer

Reply:
Yeah, good catch. I'll move it into the check.